indico/modules/events/registration/forms.py

```diff
@@ -73,6 +75,13 @@ class RegistrationFormForm(IndicoForm):
                                                                    preserve_data=True), DataRequired()],
                                                      description=_("Email addresses that will receive notifications"))
 
+    def __init__(self, *args, **kwargs):
+        super(IndicoForm, self).__init__(*args, **kwargs)
+        self.notification_sender_address.description = _('Email address set as the sender of all '
+                                                         'notifications sent to users. If empty, '
+                                                         'then {0} is used.'.format(Config.getInstance()
```

Comment:
Why not assign the email to a variable? The wrapping here doesn't look very pretty